lms/djangoapps/courseware/access.py

```diff
@@ -160,11 +178,18 @@ def _can_access_descriptor_with_start_date(user, descriptor, course_key):  # pyl
             descriptor,
             course_key=course_key
         )
-        return (
-            descriptor.start is None
-            or now > effective_start
-            or in_preview_mode()
-        )
+        if descriptor.start is None or now > effective_start or in_preview_mode():
+            return ACCESS_GRANTED
+
+    start_message = None
```

Comment:
suggestion: variable name change to something that makes it clear this is an error message.